compiler-core/src/build/package_loader/tests.rs

```diff
@@ -57,6 +57,7 @@ fn write_cache(
         accessors: Default::default(),
         unused_imports: Vec::new(),
         name_corrections: Vec::new(),
+        import_suggestions: Vec::new(),
```

Comment:
I think so, although currently the errors store a list of currently imported modules to suggest, rather than importable modules, which is what is needed for this. There are two ways I could fix this, either we keep the error message the same, and store additional data in the error, or we change the error message to something like:
```
io.println("Hello")
^^ Unknown module io.

Hint: Did you mean to import gleam/io?
```
That's just written off the top of my head, I would rewrite it to be consistent with the other error messages

Reply:
Having a suggestion in the error is a fantastic idea. We should do this for sure. We could look for a module that has the `println` member in this case.